### DISCUSSION TITLE
How to make recent directory complete with slash at the end of input?

### DISCUSSION THREAD

**Discussion by tshu-w** [marlonrichert/zsh-autocomplete#391] [Q&A] [answered]
<img width="270" alt="Screen Shot 2021-12-26 at 13 17 13" src="https://user-images.githubusercontent.com/13161779/147399604-c523a930-eeba-4081-b27d-d8d9b1862be1.png">

<img width="270" alt="Screen Shot 2021-12-26 at 13 17 50" src="https://user-images.githubusercontent.com/13161779/147399630-123ef89e-08bc-494b-8b4d-b12674331e0c.png">

**Comment by marlonrichert** [comment, marked as answer]
Press Tab (or Ctrl-I) instead of `/`. It will insert the entire path and add the `/` automatically.

**Reply by tshu-w** [reply]
Yes, I know. Sometimes I use C-e to go to the end of line and got history completion with `/`, now I have to delete `/` and press Tab

**Reply by marlonrichert** [reply]
If you want history completion, press `^R`.
